paddle/fluid/framework/parallel_executor.cc

```diff
@@ -653,11 +663,20 @@ bool ParallelExecutor::EnableParallelGraphExecution(
     }
   }
 
-  if (!member_->use_all_reduce_ || !member_->use_cuda_)
-
+  if (!member_->use_all_reduce_ || !member_->use_cuda_) {
     if (build_strategy.enable_sequential_execution_ ||
-        exec_strategy.type_ == ExecutionStrategy::ExecutorType::kExperimental)
+        exec_strategy.type_ == ExecutionStrategy::ExecutorType::kExperimental) {
       enable_parallel_graph = false;
+    }
+  }
+
+#ifdef WIN32
+  LOG_FIRST_N(WARNING, 1)
```

Comment:
Why do you use `LOG_FIRST_N ` here?

Reply:
Dont hope this bother users all the time.